elixir/apps/domain/lib/domain/auth/adapters/openid_connect/settings/changeset.ex

```diff
@@ -25,6 +25,10 @@ defmodule Domain.Auth.Adapters.OpenIDConnect.Settings.Changeset do
         {:ok, _uri} ->
           [{:discovery_document_uri, "is not a valid URL"}]
 
+        {:error, %Mint.TransportError{reason: reason}} ->
```

Comment:
This was causing a crash when typing out the discovery document URI, once you made it to `https://`